src/OrbitGl/TimeGraph.h

```diff
@@ -105,6 +105,9 @@ class TimeGraph : public orbit_gl::CaptureViewElement {
   [[nodiscard]] const orbit_client_protos::TimerInfo* FindNextFunctionCall(
       uint64_t function_address, uint64_t current_time,
       std::optional<uint32_t> thread_id = std::nullopt) const;
+  [[nodiscard]] std::vector<const orbit_client_protos::TimerInfo*> GetAllFunctionCalls(
+      uint64_t function_address) const;
```

Comment:
Do we have unit tests for TimeGraph? If so, maybe a test would be a good idea.

Reply:
Agree that it would be great if we have tests here. But we don't have test for `TimeGraph` at present.

Reply:
Okay :)